ALUM/ALUM/Components/Tags.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  Tags.swift
+//  ALUM
+//
+//  Created by Jenny Mar on 1/28/23.
+//
+
+import SwiftUI
+
+struct Tags: View {
+
+    @State var text: String = ""
+    @State var isBlue: Bool = true
+    @State var isShowing: Bool = true
+
+    var body: some View {
+        if isBlue {
+            VStack {
+                ZStack {
+                    HStack {
+
+                        Text(text)
+                            .foregroundColor(Color(.black))
+                        Spacer()
+                            .frame(width: 10)
+
+                        Button(action: {
+                            isShowing = true
+                        }, label: {
+                            Image(systemName: "xmark")
+                        })
+                        .foregroundColor(Color("NeutralGray4"))
+                        .frame(height: 8)
+                        .frame(width: 8)
+
+                    }
+
+                }
+                .padding(10)
+                .background(RoundedRectangle(cornerRadius: 100).stroke())
+                .foregroundColor(Color("Alum Light Blue"))
+                .font(.headline)
```

Comment:
font(.body) will better suit here

Checkout the table on different font types from apple
https://developer.apple.com/design/human-interface-guidelines/foundations/typography/